cmd/ocm-backplane/cloud/assume.go

```diff
@@ -0,0 +1,146 @@
+	Long: `Performs the assume role chaining necessary to generate temporary access to the customer's AWS account
+
+This command is the equivalent of running "aws sts assume-role-with-web-identity --role-arn [role-arn] --web-identity-token [ocm token] --role-session-name [email from OCM token]" behind the scenes,
+where the ocm token used is the result of running "ocm token". Then, the command makes a call to the backplane API to get the necessary jump roles for the cluster's account. It then calls the
+equivalent of "aws sts assume-role --role-arn [role-arn] --role-session-name [email from OCM token]" repeatedly for each role arn in the chain, using the previous role's credentials to assume the next
+role in the chain.
+
+This command will output sts credentials for the target role in the given cluster in formatted JSON. If no "role-arn" is provided, a default role will be used.
+`,
+	Example: `With default role:
+backplane cloud assume e3b2fdc5-d9a7-435e-8870-312689cfb29c -oenv
+
+With given role:
+backplane cloud assume e3b2fdc5-d9a7-435e-8870-312689cfb29c --role-arn arn:aws:iam::1234567890:role/read-only -oenv`,
+	Args: cobra.ExactArgs(1),
+	RunE: runAssume,
+}
+
+func init() {
+	flags := AssumeCmd.Flags()
+	flags.StringVar(&assumeArgs.roleArn, "role-arn", "", "The arn of the role for which to start the role assume process.")
+	flags.StringVarP(&assumeArgs.output, "output", "o", "env", "Format the output of the console response.")
+}
+
+type assumeChainResponse struct {
+	AssumptionSequence []namedRoleArn `json:"assumption_sequence"`
+}
+
+type namedRoleArn struct {
+	Name string `json:"name"`
+	Arn  string `json:"arn"`
+}
+
+func runAssume(_ *cobra.Command, args []string) error {
+	ocmToken, err := utils.DefaultOCMInterface.GetOCMAccessToken()
+	if err != nil {
+		return fmt.Errorf("failed to retrieve OCM token: %w", err)
+	}
+
+	roleArn := DefaultRoleArn
+	if assumeArgs.roleArn != "" {
+		roleArn = assumeArgs.roleArn
+	}
+
+	bpConfig, err := config.GetBackplaneConfiguration()
+	if err != nil {
+		return fmt.Errorf("error retrieving backplane configuration: %w", err)
+	}
+
+	initialClient, err := awsUtil.StsClientWithProxy(bpConfig.ProxyURL)
+	if err != nil {
+		return fmt.Errorf("failed to create sts client: %w", err)
+	}
+	seedCredentials, err := awsUtil.AssumeRoleWithJWT(*ocmToken, roleArn, initialClient)
+	if err != nil {
+		return fmt.Errorf("failed to assume role using JWT: %w", err)
+	}
+
+	clusterId, _, err := utils.DefaultOCMInterface.GetTargetCluster(args[0])
+	if err != nil {
+		return fmt.Errorf("failed to get target cluster: %w", err)
+	}
+
+	backplaneClient, err := utils.DefaultClientUtils.MakeRawBackplaneAPIClientWithAccessToken(bpConfig.URL, *ocmToken)
+	if err != nil {
+		return fmt.Errorf("failed to create backplane client with access token: %w", err)
+	}
+
+	// TODO Change this to the actual function we'll be using
+	response, err := backplaneClient.GetBackplaneClusterClusterId(context.TODO(), clusterId)
```

Comment:
I believe this requires https://github.com/openshift/backplane-api/pull/11 to be merged and the code autogenerated.